package.json

```diff
@@ -52,6 +54,7 @@
     "husky": "^8.0.0",
     "jest": "28.0.3",
     "prettier": "^2.3.2",
+    "faker": "^5.5.3",
```

Comment:
I strongly recommend to start using `@faker-js/faker` instead since it's now the official repository. 
Also you dont need the types :)

Reply:
Done!